src/test/java/core/basesyntax/service/impl/ParseServiceImplTest.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ParseServiceImplTest {
+    private static final String HEAD_FILE = "operation,fruit,quantity";
+    private static final String INVALID_BALANCE_STRING = "b,apple";
+    private static final int EXPECTED_LIST_SIZE = 1;
+    private static final String APPLE = "apple";
+    private static final String ORANGE = "orange";
+    private static final String VALID_BALANCE_APPLE = "b,apple,10";
+    private static final String VALID_SUPPLY_ORANGE = "s,orange,5";
+    private static final int APPLE_QUANTITY = 10;
+    private static final int ORANGE_QUANTITY = 5;
+    private static ParseServiceImpl parseService;
+    private List<FruitTransaction> resultFruit;
```

Comment:
This list is needed for every test, you can make it local variable, to not set it to null after each time.